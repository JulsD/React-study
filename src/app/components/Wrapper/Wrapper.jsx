import React from 'react';

import styles from './wrapper.css';

class Wrapper extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className = {styles.root}>
        {this.props.children}
      </div>
    );
  }

}

export default Wrapper;
