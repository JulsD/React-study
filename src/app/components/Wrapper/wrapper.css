.root {
  padding: 15px 20px;
  box-sizing: border-box;
  max-width: 75rem;
  width: 100%;
  margin: 0 auto;
}
