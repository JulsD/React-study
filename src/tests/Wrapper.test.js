import React from 'react';
import Wrapper from '../app/components/Wrapper';

test('Wrapper should render a <nav>', () => {
  const wrapper = shallow(
    <Wrapper />
  );
  expect(wrapper).toMatchSnapshot();
});
