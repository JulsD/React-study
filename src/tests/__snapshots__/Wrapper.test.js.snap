// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper should render a <nav> 1`] = `
<div
  className="root"
/>
`;
